packages/react/src/number-field/decrement/NumberFieldDecrement.test.tsx

```diff
@@ -315,4 +316,22 @@ describe('<NumberField.Decrement />', () => {
 
     expect(input).to.have.value('-2');
   });
+
+  it('prop: disabled', async () => {
```

Comment:
nit: the `it` function should describe the correct behavior. We usually place "prop: X" on the `describe` blocks.